### PR TITLE
Add BEIS logos

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Add BEIS logos and add themes and tests for them

- Also removes a couple of Welsh tests for Northern Ireland which aren't needed

### How to review 
Test new schemas look as expected with new logos

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Importing UKIS logos to eQ (2 SP)](https://trello.com/c/XH2sOlhL/5517-importing-ukis-logos-to-eq-2-sp)

File: tests/app/helpers/test_template_helpers.py [hunk 1]
```diff
@@ -216,6 +217,16 @@ def test_footer_warning_not_in_context_census_theme(app: Flask):
                 read_file("./templates/assets/images/ni-finance-mobile-logo.svg"),
             ],
         ),
+        (
+            None,
```
Comment (berroar) [inline tests/app/helpers/test_template_helpers.py:221]: Should we use the type here? 
```suggestion
            SurveyType.BEIS,
```

Comment (rmccar) [reply tests/app/helpers/test_template_helpers.py:221]: I was following the NI suff when doing this, does this mean you think we should add the equivalent for NI?

Comment (berroar) [reply tests/app/helpers/test_template_helpers.py:221]: Just not clear to me why either are `None`, the normal business config seems to have a test for both

File: scripts/run_validator.sh [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
-tag=latest
+tag=add-beis-orr-themes
```
Comment (berroar) [inline scripts/run_validator.sh:2]: Reminder to revert 👍

**Review by berroar — APPROVED** [review]
Looks good to me 👍

**Review by Yuyuutsu — APPROVED** [review]
LGTM!

**Pull request merged**